MobileCenter/MobileCenter/Support/module.modulemap

```diff
@@ -8,4 +8,5 @@ framework module MobileCenter {
   link framework "CoreTelephony"
   link framework "SystemConfiguration"
   link framework "UIKit"
+  link "sqlite3"
```

Comment:
Don't you need to update podspec as well? Keep in mind that you also need to this in multiple-platforms branch.